ansible/README.md

```diff
@@ -14,6 +14,8 @@ sudo pip install ansible==2.0.2.0
 
 #### Vagrant users
 Nothing to be done, Ansible is already installed during vagrant provisioning.
+You can skip setup and prereq steps as those have been done by vagrant for you.  
+You may jump directly to [Deploying Using CouchDB](#deploying-using-couchdb)
 
 #### Mac users
 It is assumed that a VM has been provisioned using **docker-machine**.
```

Comment:
Docker changed their default port to 2376 a while ago. Problem I see here is that the entire stack currently relies on 4243. Some scripts even have it hard-coded I believe. We also ran into this one but did not want to make that huge change here. Can we agree to make this change in a separate issue?